deepchem/models/graph_models.py

```diff
@@ -794,12 +797,18 @@ def call(self, inputs, training=False):
         gc1 = self.dropouts[i](gc1, training=training)
       gp_in = [gc1, degree_slice, membership] + deg_adjs
       in_layer = self.graph_pools[i](gp_in)
-    dense = self.dense(in_layer)
-    if self.batch_norms[-1] is not None:
-      dense = self.batch_norms[-1](dense, training=training)
-    if training and self.dropouts[-1] is not None:
-      dense = self.dropouts[-1](dense, training=training)
-    neural_fingerprint = self.graph_gather([dense, degree_slice, membership] +
+
+    startnum = len(self.graph_convs)
+    endnum = len(self.dense_layers) + len(self.graph_convs)
+    for i in range(startnum, endnum): # cont index for batchnorms and dropouts in dense layers
+        dense = self.dense_layers[i-startnum](in_layer)
+        if self.batch_norms[i] is not None:
+            dense = self.batch_norms[i](dense, training = training)
```

Comment:
Can you add `batch_normalize_kwargs` as in `WeaveModel` so users can configure their batch norms? https://github.com/deepchem/deepchem/blob/master/deepchem/models/graph_models.py#L96